### ISSUE TITLE
translation missing: zh-TW.activerecord.errors.messages.array.subset_of

### ISSUE THREAD

**Issue by khiav223577** [infinum/array_validator#6]
I got this error message when one of items in array is not in the subset passed to `validates`.
It seems that I have to add some translations to i18n files, but there is no documentation of what have to be added.

**Comment by vr4b4c** [comment]
Hi,

The library includes a set of default [error translations](https://github.com/infinum/array_validator/blob/master/lib/locale/en.yml) in English. You are free to copy those translations and adapt to the desired language.

**Comment by khiav223577** [comment]
Thanks. It helps.

It would be nicer to add this information in  `README.md` :)

**Comment by vr4b4c** [comment]
Agreed, I'll keep this issue open until the translation section is added into `README.md`.

PR is welcome.
